nodewatcher/nodewatcher.py

```diff
@@ -142,6 +131,12 @@ def maintainSize(region, asg):
         log.debug('capacity less then or equal to min size.')
         return True
 
+
+def saveIdleTime(persisted_data):
+    with open('/var/run/nodewatcher/node_idletime.json', 'w') as outfile:
```

Comment:
At the start of the nodewatcher you should check the dir is there and if it's not then create it.